modules/android/pages/vector-search-api-reference.adoc

```diff
@@ -0,0 +1,286 @@
+= Vector Search API Reference
+:page-aliases: 
+ifdef::show_edition[:page-edition: {release}]
+ifdef::prerelease[:page-status: {prerelease}]
+:page-role:
+:description: API reference for Vector Search in Couchbase Lite.
+:keywords: edge AI api swift ios macos apple vector search generative
+
+== Vector Index API
+
+You can use the Vector Index API's core functionality to efficiently perform similarity search at scale on Vector Indexes.
+You can make use of new qurry functions such as `vector_match()` to query indexed vectors.
+
+=== Vector Encoding
+
+Vector encoding is the process of representing data points such as images, words, or documents as vectors in high-dimensional space where each dimension represents a different attribute of the data. 
+Encoding is a crucial step in preparing data for similarity or nearest neighbour search operations.
+
+Couchbase Lite support three encoding types:
```

Comment:
There appears to be some confusion around vector encoding. Please catchup with Iveta on this
What you describe here is how Couchbase Lite vector indexes are encoded. 

At a high level, this is what happens
- Data is translated to vector of floats
- Vector index is created on the vectors. You use encoding to save space when storing indexes.